### PR TITLE
Reorder to put identity bindings first

### PR DESCRIPTION
**Pull request by martinthomson**
I did some reframing to justify this organization.  This probably needs
another review.

Attn. @adamroach - [preview of complete draft](https://martinthomson.github.io/sdp-uks/reorder/draft-ietf-mmusic-sdp-uks.html) because the [diff](https://tools.ietf.org/rfcdiff?url1=https://martinthomson.github.io/sdp-uks/draft-ietf-mmusic-sdp-uks.txt&url2=https://martinthomson.github.io/sdp-uks/reorder/draft-ietf-mmusic-sdp-uks.txt) is pretty hard to follow.

### CODE REVIEW

File: draft-ietf-mmusic-sdp-uks.md [hunk 1]
```diff
@@ -66,104 +68,87 @@ techniques are defined for each.
...
+The attacker obtains an identity assertion for an identity it controls, but
+binds that to the fingerprint of one peer.  The attacker is then able to cause a
+TLS connection to be established where two endpoints communicate.  The victim
+that has its fingerprint copied by the attack correctly believes that it is
+communicating with the other victim; however, the other victim incorrectly
+believes that it is communicating with the attacker.
 
-This attack applies to any communications established based on the SDP
-`fingerprint` attribute {{!RFC8122}}.
+A similar attack can be mounted without to any communications established based
```
Comment (adamroach) [inline draft-ietf-mmusic-sdp-uks.md:141]: Nit: s/without to any/without any/

File: draft-ietf-mmusic-sdp-uks.md [hunk 2]
```diff
@@ -273,50 +448,17 @@ conduct two sessions concurrently, if the attacker (Mallory) is on the network
 path between the victims, and if the same certificate - and therefore SDP
```
Comment (adamroach) [inline draft-ietf-mmusic-sdp-uks.md:448]: > This attack creates an asymmetry in the beliefs about the identity of peers. However, this attack is only possible if the victim (Norma) is willing to conduct two sessions concurrently

Unless I'm mistaken, the precondition here is even more difficult than this phrasing would imply. Assuming my understanding is correct, I believe the following wording would be more accurate: `...is only possible if the victim (Norma) can be compelled to initiate two concurrent sessions nearly simultaneously...`

_Don't treat this comment as blocking. If you agree, make the change; if you don't, discard it and move the document forward._

File: draft-ietf-mmusic-sdp-uks.md [hunk 3]
```diff
@@ -349,21 +491,21 @@ identifier.
```
Comment (adamroach) [inline draft-ietf-mmusic-sdp-uks.md:491]: > This document only defines use of this extensions for SDP

Nit: "...this extension..."

**Review by adamroach — APPROVED** [review]

**Comment by martinthomson** [comment]
Thanks Adam!
